### PR TITLE
feat(sdk+openai): support base64 images upload

### PR DESCRIPTION
**Pull request by galkleinman**

### CODE REVIEW

File: packages/sample-app/sample_app/openai_vision_base64_example.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+api_key = os.environ.get("OPENAI_API_KEY")
+
+
+def encode_image(image_path):
+    with open(image_path, "rb") as image_file:
+        return base64.b64encode(image_file.read()).decode('utf-8')
+
+
+image_path = "/Users/galklm/Downloads/elephant.jpeg"
```
Comment (nirga) [inline packages/sample-app/sample_app/openai_vision_base64_example.py:16]: nit: maybe add an image together with the sample app and point to it here so people can actually run the sample app and see it in action?

File: packages/traceloop-sdk/tests/test_image_uploader.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import nest_asyncio
```
Comment (nirga) [inline packages/traceloop-sdk/tests/test_image_uploader.py:1]: fyi: I think it's an unmaintained repo that has been archived recently:
https://github.com/erdewit/nest_asyncio

**Review by nirga — APPROVED** [review]

File: packages/traceloop-sdk/tests/test_image_uploader.py (hunk 2 above)
Comment (galkleinman) [reply packages/traceloop-sdk/tests/test_image_uploader.py:1]: the issue is that is solves an issue with a test i added... look at the last commit (or the one before), I specifically added it to the tests to resolve an issue.

File: packages/sample-app/sample_app/openai_vision_base64_example.py (hunk 1 above)
Comment (galkleinman) [reply packages/sample-app/sample_app/openai_vision_base64_example.py:16]: will do

**Pull request merged**